go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/ipfs/iptb-plugins
```

Comment:
Change this to `github.com/TRON-US/iptb-plugins`, otherwise, the tagged release won't work. actually I suggest changing to `github.com/TRON-US/bttb-plugins`, or if you want to use the original iptb all the way, then it's `github.com/TRON-US/iptb-btfs-plugins`

In fact, once you do, you need to refresh all the paths inside this repository.